src/ServiceControl.MultiInstance.AcceptanceTests/TestSupport/ServiceControlComponentRunner.cs

```diff
@@ -271,8 +272,14 @@ async Task InitializeServiceControlAudit(ScenarioContext context, int instancePo
                 typeof(ServiceControlComponentRunner).Assembly.GetName().Name
             };
 
+            ConfigurationManager.AppSettings.Set("ServiceControl.Audit/PersistenceType", typeof(Audit.Persistence.RavenDb.RavenDbPersistenceConfiguration).AssemblyQualifiedName);
```

Comment:
Why not in-memory for the multi-instance tests?

Reply:
I tried, but we have some tests that assume raven35 is in use.

I agree that we should use InMemory but I wanted to get the tests green as a first step